src/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/Computer.cs

```diff
@@ -20,9 +20,6 @@ public class ComputerPrincipal : AuthenticablePrincipal
         //
         public ComputerPrincipal(PrincipalContext context) : base(context)
         {
-            if (context == null)
-                throw new ArgumentException(SR.NullArguments);
-
```

Comment:
Just to verify, you deleted this as dead code because the base ctor does the same check?